5.2.6 Dependency conflict with juice@11.0.3 causing runtime errors

**Environment:**
- `@maizzle/framework`: `5.2.6`
- Package manager: Bun `1.2.23`

**Description:**

After upgrading to `@maizzle/framework@5.2.6`, which bumped `juice` from `10.0.1` to `11.0.3`, we're experiencing runtime errors in production environments:

```
error: Cannot find module 'entities/lib/decode.js' from '/path/to/project/shared/node_modules/posthtml-parser/node_modules/htmlparser2/lib/Tokenizer.js'
```

**Root Cause:**

The issue is caused by a transitive dependency conflict in `juice@11.0.3`:

1. `juice@11.0.3` depends on `web-resource-inliner@^7.0.0`
2. `web-resource-inliner@7.0.0` depends on `htmlparser2@^5.0.0`
3. `htmlparser2@5.0.0` depends on `entities@^2.0.0`
4. Meanwhile, other parts of the dependency tree (like `posthtml-parser`) use `htmlparser2@9.1.0` which requires `entities@^4.5.0`

The problem is that `entities@4.x` changed its directory structure from `lib/` to `dist/commonjs/`, breaking compatibility with older versions of `htmlparser2@5.0.0` that still try to import from the old `entities/lib/decode.js` path.

**Package Manager Behavior:**

When the package manager resolves dependencies, it installs `entities@4.x` (required by the newer `htmlparser2@9.1.0`), but `htmlparser2@5.0.0` (used by `web-resource-inliner`) still tries to import from the old path which no longer exists, causing runtime errors.

**Suggested Solution:**

**Consider reverting to `juice@10.x` until the `juice` maintainers resolve the dependency conflict** in their `web-resource-inliner` dependency.

The `juice@10.0.1` version used in `@maizzle/framework@5.2.5` doesn't have this issue because it uses older, compatible versions of the dependency chain.

Would it be possible to temporarily pin `juice` to `10.x` until this upstream issue is resolved?

Thank you!

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@maizzle/framework",
-  "version": "5.2.6",
+  "version": "5.2.7",
   "description": "Maizzle is a framework that helps you quickly build HTML emails with Tailwind CSS.",
   "license": "MIT",
   "type": "module",
@@ -63,7 +63,7 @@
     "html-crush": "^6.0.19",
     "is-url-superb": "^6.1.0",
     "istextorbinary": "^9.5.0",
-    "juice": "^11.0.3",
+    "juice": "^10.0.1",
     "lodash-es": "^4.17.21",
     "morphdom": "^2.7.4",
     "ora": "^8.1.0",

